setup.py

```diff
@@ -14,7 +14,9 @@
             'matplotlib',
             'jpype1==0.6.3',
             'scipy',
-            'sklearn'],
+            'sklearn',
+            'netCDF4',
+            'datetime'],
```

Comment:
`datetime` is built-in, so it does not need to be listed here as a dependency

Reply:
Thanks for the reminder. I didn't notice this before.